OpenRA.Game/Primitives/BitSet.cs

```diff
@@ -98,7 +98,7 @@ public override string ToString()
 		public static bool operator !=(BitSet<T> me, BitSet<T> other) { return !(me == other); }
 
 		public bool Equals(BitSet<T> other) { return other == this; }
-		public override bool Equals(object obj) { return obj is BitSet<T> && Equals((BitSet<T>)obj); }
+		public override bool Equals(object obj) { return obj is BitSet<T> bitSets && Equals(bitSets); }
```

Comment:
```suggestion
		public override bool Equals(object obj) { return obj is BitSet<T> bitSet && Equals(bitSet); }
```
?
(This is only a single `BitSet` isn't it.)